app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/ContainerFragment.kt

```diff
@@ -0,0 +1,61 @@
+package com.sun_asterisk.moviedb_50.screen.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentTransaction
+import com.sun_asterisk.moviedb_50.R
+
+class ContainerFragment : Fragment() {
+    private var startingPosition = 0;
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_container, container, false)
+        loadFragment(HomeFragment(), startingPosition)
+        return view
+    }
+    private fun loadFragment(
+        fragment: Fragment?,
+        position: Int
+    ): Boolean {
+        if (fragment != null) {
```

Comment:
```suggestion
        fragment?: return false 
```
có nghĩa laf: nếu fragment == null return false 
hoặc 
```suggestion
        fragment?.let {
```
có nghĩa là là nếu frament != null thì ...
viết theo cách đầu cho gọn